api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -191,6 +203,63 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     return ResponseEntity.ok(createdWorkspace);
   }
 
+  @Override
+  public ResponseEntity<WorkspaceOperation> createWorkspaceAsync(Workspace workspace) {
+    // Basic request validation.
+    validateWorkspaceApiModel(workspace);
+    getLiveCdrVersionId(workspace.getCdrVersionId());
+
+    // TODO: enforce access level check here? Not strictly necessary, but may make sense as
```

Comment:
I don't see any easy existing method to call for doing this - it can probably wait. My thinking here is that if the logic for `processCreateWorkspaceTask` ever changes down the road, we'd want to ensure there aren't any assumptions of access downstream.